packages/bpk-tokens/src/common/base/animations.json

```diff
@@ -0,0 +1,7 @@
+{
+  "aliases": {
+    "ANIMATION_DURATION_XS": "50ms",
```

Comment:
just a quick note on this, these values have been discussed or are just arbitrary values?

Reply:
They are already used on web, and I just added them to the RN tokens too.
This is probably something we need to discuss / get some designer input on